.ci/docker/ci_commit_pins/executorch.txt

```diff
@@ -1 +1 @@
-d4b3e5cc607e97afdba79dc90f8ef968142f347c
+2d35b304c1afd0a37f6cdc6b00aa767ea9b81a11
```

Comment:
while you're at it, can you change this to latest? 

172574a6be5910a4609e4ed1bef2b6b8475ddb3d

It contains this: https://github.com/pytorch/executorch/pull/4007